rts/builds/legacy/CMakeLists.txt

```diff
@@ -68,13 +68,15 @@ list(APPEND engineLibraries pr-downloader)
 list(APPEND engineLibraries RmlCore)
 list(APPEND engineLibraries RmlDebugger)
 list(APPEND engineLibraries RmlSolLua)
+list(APPEND engineLibraries lunasvg)
 
 ### Assemble external incude dirs
 list(APPEND engineIncludes ${OPENAL_INCLUDE_DIR})
 list(APPEND engineIncludes ${ENGINE_SRC_ROOT_DIR}/lib/asio/include)
 list(APPEND engineIncludes ${ENGINE_SRC_ROOT_DIR}/lib/slimsig/include)
 list(APPEND engineIncludes ${ENGINE_SRC_ROOT_DIR}/lib/cereal/include)
 list(APPEND engineIncludes ${ENGINE_SRC_ROOT_DIR}/lib/RmlSolLua/src)
+list(APPEND engineIncludes ${ENGINE_SRC_ROOT_DIR}/lib/lunasvg/include)
```

Comment:
why is this needed? Does engine reference this include directly? Same for the linking of lunasvg, isn't that done on Rmlui side? Does engine reference it's interface directly?